backend/routes/userEndpoints.js

```diff
@@ -67,4 +98,11 @@ userEndpoints.route("/:id").delete((req, response) => {
   });
 });
 
+function checkPassword(userInput, dbPassword) {
```

Comment:
I made the comment above before seeing this. Can't we add this above to lessen the code?